indico/util/string.py

```diff
@@ -190,18 +190,23 @@ def remove_tags(text):
     return remove_extra_spaces(pattern.sub(' ', text))
 
 
-def render_markdown(text, **kwargs):
+def render_markdown(text, escape_latex_math=True, **kwargs):
     """ Mako markdown to html filter """
-    math_segments = []
-    placeholder = u"\uE000"
+    if escape_latex_math:
+        math_segments = []
+        placeholder = u"\uE000"
 
-    def math_replace(m):
-        math_segments.append(m.group(0))
-        return placeholder
+        def _math_replace(m):
+            math_segments.append(m.group(0))
+            return placeholder
+
+        text = re.sub(r'\$[^\$]+\$|\$\$(^\$)\$\$', _math_replace, to_unicode(text))
```

Comment:
OTOH, this looks exactly the same as https://github.com/indico/indico/pull/1881/files#diff-b0098f64f333656ff7eb7ffbc29af757R714